src/components/notifications/Discord.vue

```diff
@@ -53,13 +53,24 @@
             </div>
         </div>
     </div>
+
+    <div class="mb-3">
+        <label for="discord-disable-url" class="form-label">{{ $t("Disable URL in Notification") }}</label>
+        <div class="form-check form-switch">
+            <input id="discord-disable-url" v-model="$parent.notification.disableUrl" class="form-check-input" type="checkbox">
+            <label class="form-check-label" for="discord-disable-url">{{ $t("Disable URL") }}</label>
+        </div>
```

Comment:
having something labeled twice (see the duplicate `<label for=`) is not quite so nice for people using assistive technologies.

Please adjust the translation key accordingly

```suggestion
        <div class="form-check form-switch">
            <input id="discord-disable-url" v-model="$parent.notification.disableUrl" class="form-check-input" type="checkbox">
            <label class="form-check-label" for="discord-disable-url">{{ $t("Disable URL in Notification") }}</label>
        </div>
```